src/examples/button/IconButtonSizes.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Row } from '@zendeskgarden/react-grid';
+import { IconButton } from '@zendeskgarden/react-buttons';
+import { ReactComponent as ZendeskIcon } from '@zendeskgarden/svg-icons/src/26/zendesk.svg';
+
+const Example = () => (
+  <Row justifyContent="around">
+    <IconButton size="small">
+      <ZendeskIcon />
+    </IconButton>
+    <IconButton>
```

Comment:
Use explicit `size="medium"` here for clarity and consistency with other examples.